rokomari_c/urls.py

```diff
@@ -19,5 +19,6 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('api/', include('home.urls')),
```

Comment:
@shahriaralamsakib  We should have separate url files for each app. Those urls file will be linked in the main urls file